bindings/netstandard/ElectionGuard/ElectionGuard.Decryption/ChallengeResponse/ValidateChallengeResponse.cs

```diff
@@ -0,0 +1,98 @@
+using ElectionGuard.Guardians;
+
+namespace ElectionGuard.Decryption.ChallengeResponse;
+
+/// <summary>
+/// functions to validate the challenge responses.
+/// </summary>
+public static class ValidateChallengeResponseExtensions
+{
+    /// <summary>
+    /// Compute the commitment for the selection using 
+    /// the publically available data.
+    ///
+    /// equations (63) and (64) in the spec
+    /// </summary>
+    public static ElGamalCiphertext ComputeCommitment(
+        this SelectionChallengeResponse self,
+        ElGamalCiphertext ciphertext,
+        ElementModQ challenge,
+        Dictionary<string, ElectionPublicKey> guardians,
+        ElementModP m_i)
+    {
+        return self.ComputeCommitment(
+            ciphertext.Pad,
+            challenge,
+            guardians,
+            m_i);
+    }
+
+    /// <summary>
+    /// Compute the commitment for the selection using 
+    /// the publically available data.
+    ///
+    /// equations (63) and (64) in the spec
+    /// </summary>
+    public static ElGamalCiphertext ComputeCommitment(
+        this SelectionChallengeResponse self,
+        ElGamalCiphertext ciphertext,
+        ElementModQ challenge,
+        ElementModP commitmnetOffset,
+        ElementModP m_i)
+    {
+        return self.ComputeCommitment(
+            ciphertext.Pad,
+            challenge,
+            commitmnetOffset,
+            m_i);
+    }
+
+    /// <summary>
+    /// Compute the commitment for the selection using 
+    /// the publically available data.
+    ///
+    /// equations (63) and (64) in the spec
+    /// </summary>
+    public static ElGamalCiphertext ComputeCommitment(
+        this SelectionChallengeResponse self,
+        ElementModP ciphertextPad,
+        ElementModQ challenge,
+        Dictionary<string, ElectionPublicKey> guardians,
+        ElementModP m_i)
+    {
+        // Π 𝐾^𝑖^m mod 𝑝
+        var commitmentOffset = guardians.ComputeCommitmentOffset(self.SequenceOrder);
```

Comment:
note this will only work with all available guardians. i left it for testing and so,

TODO: mark this obsolete and make it take in the ciphertext election context so we can throw an error if the dictionary does not contain all guardians